.cirrus.yml

```diff
@@ -16,9 +16,9 @@ env:
   # Maximum cache period (in weeks) before forcing a new cache upload.
   CACHE_PERIOD: "2"
   # Increment the build number to force new mambaforge cache upload.
-  MAMBA_CACHE_BUILD: "1"
+  MAMBA_CACHE_BUILD: "0"
   # Increment the build number to force new mint cache upload.
-  MINT_CACHE_BUILD: "1"
+  MINT_CACHE_BUILD: "0"
```

Comment:
@pletzer BTW the purpose of the `MAMBA_CACHE_BUILD` and `MINT_CACHE_BUILD` variables is a way to manually for either of the caches on `cirrus-ci` to purge their cache and start afresh. To do this simply bump the number up.

As it stands, both of these caches are automatically purged and rebuilt every `CACHE_PERIOD` weeks, which you can change to whatever you chose.

The point here is that you don't want to be using the same cache for too long as you'll miss package updates from the `conda-forge` community, so you don't want to store up technical debt. However, you want benefit from using a cache to speed-up the time it takes for a pull-request to run.